### PR TITLE
feat: computed-types resolver

### PR DESCRIPTION
**Pull request by jorisre**
Welcome to the [computed-types](https://github.com/neuledge/computed-types) resolver ! 🎉 
Related to https://github.com/react-hook-form/resolvers/issues/161

👀 :
- Update repository dependencies
- Only `async` mode for that new resolver
- No `criteriaMode: all` because computed-types returns only the first error of each field
- Basic `type`, always `ValidationError` :/ errors don't have a property such as a type, code or a key

### CODE REVIEW

File: computed-types/src/__tests__/__snapshots__/computed-types.ts.snap [hunk 1]
```diff
@@ -0,0 +1,65 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`computedTypesResolver should return a single error from computedTypesResolver when validation fails 1`] = `
+Object {
+  "errors": Object {
+    "birthYear": Object {
+      "message": "Expect value to be \\"number\\"",
+      "ref": undefined,
+      "type": "ValidationError",
```
Comment (jorisre) [inline computed-types/src/__tests__/__snapshots__/computed-types.ts.snap:9]: Always same, feel free to suggest a better way

File: computed-types/src/computed-types.ts [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+import type { ValidationError } from 'computed-types';
+import type { Resolver } from './types';
+
+const parseErrorSchema = (
+  computedTypesError: ValidationError,
+  parsedErrors: FieldErrors = {},
+  path = '',
+) => {
+  return (computedTypesError.errors || []).reduce((acc, error) => {
+    const _currentPath = String(error.path[0]);
```
Comment (jorisre) [inline computed-types/src/computed-types.ts:12]: path is always an array of one item

File: computed-types/src/types.ts [hunk 3]
```diff
@@ -0,0 +1,14 @@
+import type {
+  FieldValues,
+  ResolverResult,
+  UnpackNestedValue,
+  ResolverOptions,
+} from 'react-hook-form';
+
+export type Resolver = (
+  schema: any,
```
Comment (jorisre) [inline computed-types/src/types.ts:9]: Can't find a good way to type the schema.

@kotarella1110 if you want to play with that one, _(no worries if you haven't time :))_

File: nope/src/types.ts [hunk 4]
```diff
@@ -4,7 +4,7 @@ import type {
   ResolverResult,
   UnpackNestedValue,
 } from 'react-hook-form';
-import type NopeObject from 'nope-validator/lib/cjs/NopeObject';
+import type { NopeObject } from 'nope-validator/lib/cjs/NopeObject';
```
Comment (jorisre) [inline nope/src/types.ts:7]: Nope V1 update

**Review by bluebill1049 — APPROVED** [review]
LGTM. Thank you @jorisre

**Pull request merged**